profiles/base/package.use.mask

```diff
@@ -683,3 +683,8 @@ net-proxy/squid ipf-transparent pf-transparent
 # Alexis Ballier <aballier@gentoo.org> (2011-02-16)
 # Win32 specific useflag for vlc. Can be used for cross-compiling.
 media-video/vlc directx
+
+# Ross Charles Campbell <rossbridger.cc@gmail.com> (2020-09-21)
+# Since dev-games/ogre:0/1.12 is masked, mask contribovis USE to
```

Comment:
Really nit picking and it's for your information rather than needing to fix it per-se: it's the version that's masked, not the subslot.